GameServerCore/Enums/SpellDataFlags.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace GameServerCore.Enums
+{
+    /// <summary>
+    /// Enumerator containing all(?) possible spell flags found in spell data. Used in determining how a spell functions.
+    /// </summary>
+    /// TODO: Verify if this is finished.
+    [Flags]
+    public enum SpellDataFlags
+    {
+        AutoCast = 1 << 1,
+        InstantCast = 1 << 2,
+        PersistThroughDeath = 1 << 3,
+        NonDispellable = 1 << 4,
+        NoClick = 1 << 5,
+        AffectImportantBotTargets = 1 << 6,
+        AllowWhileTaunted = 1 << 7,
+        NotAffectZombie = 1 << 8,
+        AffectUntargetable = 1 << 9,
+        AffectEnemies = 1 << 10,
+        AffectFriends = 1 << 11,
+        AffectNeutral = 1 << 14,
+        // 4000 + C00: 4C00
+        AffectAllSides = AffectNeutral + 3 * (1 << 10),
+        AffectBuildings = 1 << 12,
+        AffectMinions = 1 << 15,
+        AffectHeroes = 1 << 16,
+        AffectTurrets = 1 << 17,
+        // 20000 + 18000: 38000
+        AffectAllUnitTypes = AffectTurrets + 3 * (1 << 15),
```

Comment:
Is it the same as `AffectTurrets + AffectHeroes + AffectMinions `?

Reply:
Yes.